cockroachcloud/serverless-faqs.md

```diff
@@ -148,7 +148,7 @@ We don't recommend `userfile` for `EXPORT` operations. You can either add billin
 
 ### Is change data capture available to me?
 
-Yes, {{ site.data.products.serverless-plan }} clusters have access to [Core Changefeeds](../{{site.versions["stable"]}}/changefeed-examples.html#create-a-core-changefeed).
+Yes, {{ site.data.products.serverless-plan }} clusters have access to [Core Changefeeds](../{{site.versions["stable"]}}/changefeed-examples.html#create-a-core-changefeed). Creating {{ site.data.products.enterprise }} changefeeds on {{ site.data.products.serverless }} is supported with an [{{ site.data.products.enterprise }} license](../v22.1/enterprise-licensing.html).
```

Comment:
I thought all serverless clusters had enterprise licenses? cc @emily-horing

Reply:
I'll wait for Emily to confirm, but the free forever serverless clusters I imagine would not have an enterprise license.

Reply:
Yes, all serverless clusters have access to enterprise features

Reply:
Thanks for confirming. I'll fix.